README.md

```diff
@@ -15,51 +15,105 @@ When you are done:
 #### Loops & Lists
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
-# copy & paste your solution here
+def return_max(l):
+  if len(l)== 0:
+    return None
+  return max(l)
+def return_min(l):
+  if len(l)== 0:
+    return None
+  return min(l)
 ```
 
 - [Min Value](https://repl.it/@Admin7/minvalue)
 ```python
-# copy & paste your solution here
+def return_min(l):
+  if len(l)== 0:
+    return None
+  return min(l)
 ```
 
 - [Extend](https://repl.it/@Admin7/extendlist)
 ```python
-# copy & paste your solution here
+for i in iter:
+    l.append(i)
+  return(l)
 ```
 
 - [Extend](https://repl.it/@Admin7/creatematrix)
 ```python
-# copy & paste your solution here
+def create_matrix(h, w):
+  # a list to hold the list
+  aList = []
+  for i in range(h):
+    #make a list for each row
+    bList = []
+    for x in range(w):
+      if w == 1:
+        break
+      else:
+        bList.append('#')
+    aList.append(bList)
+  return aList
 ```
 
 
 #### Dictionaries
 - [Receipt Tally](https://repl.it/@Admin7/receipttally)
 ```python
-# copy & paste your solution here
+def get_total(h, w):
```

Comment:
This was my mistake but `h` and `w` probably aren't great parameter names, right? Don't be afraid to rename these so they work for you!